tests/test_SklearnRandomForestConverters.py

```diff
@@ -5,13 +5,14 @@
 # --------------------------------------------------------------------------
 
 import unittest
+import numpy
 from sklearn.ensemble import RandomForestClassifier
 from sklearn.ensemble import RandomForestRegressor
 from sklearn.ensemble import ExtraTreesClassifier
 from sklearn.ensemble import ExtraTreesRegressor
-from skl2onnx.common.data_types import onnx_built_with_ml
+from skl2onnx.common.data_types import onnx_built_with_ml, FloatTensorType
 from test_utils import dump_one_class_classification, dump_binary_classification, dump_multiple_classification
-from test_utils import dump_multiple_regression, dump_single_regression
+from test_utils import dump_multiple_regression, dump_single_regression, convert_model, dump_data_and_model
```

Comment:
We don't have pep8 check on tests folder yet, still I would recommend running pep8 check locally using flake8. This line is clearly > 79 characters.

Reply:
I will change this line to follow pep8. I saw the preceding line was even longer so I didn't think twice about adding to this line. Will be nice if we can get the unit tests all to pep8 and start enforcing on them too. I see https://github.com/onnx/sklearn-onnx/pull/76 has done just that among other things.